src/compressed_tensors/quantization/quant_args.py

```diff
@@ -234,7 +276,11 @@ def validate_model_after(model: "QuantizationArgs") -> Dict[str, Any]:
 
     def pytorch_dtype(self) -> torch.dtype:
         if self.type == QuantizationType.FLOAT:
-            return FP8_DTYPE
+            if self.num_bits == 8:
+                return FP8_E4M3_DATA.dtype
+            else:
+                assert self.num_bits == 4
+                raise NotImplementedError("Not supported for FP4")
```

Comment:
Might help readability to break this up a bit
```suggestion
            elif self.num_bits == 4:
                raise NotImplementedError("Not yet supported for FP4")
            else:
                raise NotImplementedError("Only 4-bit or 8-bit allowed")
```